packages/lisk-transactions/src/utils/validation/schema.ts

```diff
@@ -103,7 +103,7 @@ export const baseTransaction = {
 		},
 		timestamp: {
 			type: 'integer',
-			minimum: 0,
+			minimum: -2147483648,
```

Comment:
Does it mean that we always accept transactions with negative timestamps or do we allow negative timestamps for a few transactions which are exceptions in the blockchain?

Reply:
It seems like there is no exception and allowing negative number per current implementation.
https://github.com/LiskHQ/lisk/blob/development/framework/src/modules/http_api/schema/swagger.yml#L1278-L1283
https://github.com/LiskHQ/lisk/blob/development/framework/src/modules/chain/logic/transaction.js#L688-L696